src/test/java/com/stripe/net/LiveStripeResponseGetterTest.java

```diff
@@ -139,7 +170,7 @@ public void testAppInfo() {
 
 		Stripe.setAppInfo("MyAwesomePlugin", "1.2.34", "https://myawesomeplugin.info");
 
-		Map<String, String> headers = srg.getHeaders(options);
+		Map<String, String> headers = LiveStripeResponseGetter.getHeaders(options);
```

Comment:
Opportunistic warning fix: `LiveStripeResponseGetter.getHeaders` is a static method and so should not be accessed from an instance.

Reply:
Yep. Thanks for the comment.